### PR TITLE
Making the license.txt available and understandable

### PR DESCRIPTION
**Pull request by mgifford**
Folks should know that they shouldn't need to worry. We're just implementing a best practice here.

### CODE REVIEW

File: LICENSE.md [hunk 1]
```diff
@@ -15,6 +15,10 @@ Copyright © 2008 W3C® (MIT, ERCIM, Keio), All Rights Reserved. W3C liability,
 
 Copyright © 2017-2018 W3C® (MIT, ERCIM, Keio, Beihang). W3C liability, trademark and document use rules apply.
 
+### Software used by this project
+
+We have used other open source software libraries to build this project. For those who are interested we have listed these and the [exact license that they use here](https://github.com/GSA/open-product-accessibility-template/tree/main/license).
```
Comment (dmundra) [inline LICENSE.md:20]: @mgifford could the link be relative.

**Review by dmundra — CHANGES_REQUESTED** [review]
Looks good. Small link change recommendation.

File: LICENSE.md (hunk 1 above)
Comment (mgifford) [reply LICENSE.md]: Hopefully, it's all good now.

**Review by dmundra — APPROVED** [review]
Yes! Thank you!

**Pull request merged**